lib/yard/templates/helpers/html_helper.rb

```diff
@@ -89,6 +89,9 @@ def html_markup_markdown(text)
                              :tables,
                              :with_toc_data,
                              :no_intraemphasis).to_html
+        when 'Commonmarker'
+          # GFM configs are on by default; use YARD for syntax highlighting
+          Commonmarker.to_html(text, plugins: {syntax_highlighter: nil})
```

Comment:
```suggestion
          Commonmarker.to_html(text)
```
However, preference for the provider’s highlighting is outside of this PR’s scope.